src/gui/tray/TextButtonContents.qml

```diff
@@ -0,0 +1,22 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
```

Comment:
same comment about minor versions of qml imports

Reply:
Upgraded all to x.15